kolibri/core/auth/migrations/0004_auto_20170816_1607.py

```diff
@@ -7,69 +7,6 @@
 from kolibri.core.auth.constants.role_kinds import ADMIN
 
 
-def device_owner_to_super_user(apps, schema_editor):
```

Comment:
Deleting this migration would drop support for upgrading from 0.6.0 and below. I think this is OK, but I think it's useful for us to be explicit that this is happening.

Reply:
The other consideration here is that if we are preventing this upgrade path, it actually gives us license to make more aggressive edits to the migrations - we could prevent the DeviceOwner table from ever being created in the first place.

Reply:
I just wanted to remove the code that was causing problems in the tests, to be as less intrusive as possible. Of course, having the DeviceOwner table in the previous migrations does not make sense anymore from a code point of view, so I'm happy to delete it :)

Reply:
Looks like we need to remove references to it from the tests for it too: `kolibri/core/auth/test/test_data_migrations.py`

Reply:
Any problem in removing totally this test?
I don't see the point of it if the migration between models does not exist any more and it's gonna be really hard to make it pass because using `DevicePermissions.objects.create(user=deviceowner, is_superuser=True)` requires the user to be saved, but the user can't be saved at that moment of the migrations because `FacilityUser.objects.create` adds `gender` or `birth_year` fields that can't be saved as they didn't exist in the db yet.